proto/minder/v1/minder.proto

```diff
@@ -78,6 +78,16 @@ service ArtifactService {
             auth_scope: OBJECT_OWNER_PROJECT
         };
     }
+
+    rpc GetArtifactByName (GetArtifactByNameRequest) returns (GetArtifactByNameResponse) {
+        option (google.api.http) = {
+            get: "/api/v1/artifact/{name}"
```

Comment:
Do we have to worry about names and IDs colliding?  If so, we could add some additional text to this one to ensure it doesn't conflict.

Reply:
I think the expected format for `--name` is `repoOwner/repoName/artifactName` which should not allow for such conflicts, right?

Reply:
yes but I think Evan has a point and I was just being sloppy. Let me change the path

Reply:
@evankanderson @rdimitrov one more patch in this PR, WDYT?

Reply:
Ah, I was too fast to reply to this one. Got it now 👍

Reply:
thans for the ack, I'm going to wait for Evan to see if this is what he meant

Reply:
actually it seems like folks would like to do a release, do I'm going to go YOLO and just merge this.